samples/nrf9160/https_simple/prj.conf

```diff
@@ -0,0 +1,14 @@
+CONFIG_BSD_LIBRARY=y
```

Comment:
Could we create sections in the project file, as some other samples do:
https://github.com/NordicPlayground/fw-nrfconnect-nrf/blob/master/samples/nrf9160/mqtt_simple/prj.conf?

Such organized project files are much easier to read.